src/Tribe/Service_Providers/Customizer.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Handles the customizer CSS overrides from TEC
+ *
+ * @since   TBD
+ * @package Tribe\Tickets\Service_Providers
+ */
+
+namespace Tribe\Tickets\Service_Providers;
+
+use Tribe__Customizer;
+use Tribe__Utils__Color;
+
+class Customizer extends \tad_DI52_ServiceProvider {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Customizer

<sub>hash: 2d89a99dc5e154a97ae5c65efe5129c9</sub>